cost/azure/savings_realized/azure_savings_realized.pt

```diff
@@ -7,7 +7,7 @@ category "Cost"
 severity "low"
 default_frequency "monthly"
 info(
-  version: "3.7",
+  version: "3.8",
   provider: "Azure",
   service: "Compute",
   policy_set: "Savings Realized"
```

Comment:
Regarding the parameter on Line 44 - are you able to add a 'default' field to this parameter?

For example: `default "2022-10"`